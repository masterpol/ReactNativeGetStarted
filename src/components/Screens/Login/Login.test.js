import React from 'react'
import Login from './Login'

describe('Login Screen', () => {
  let props

  beforeEach(() => {
    props = {}
  })

  describe('structure', () => {
    it('should match the snapshot', () => {
      const wrapper = render(<Login {...props} />)
      expect(wrapper).toMatchSnapshot()
    })
  })
})