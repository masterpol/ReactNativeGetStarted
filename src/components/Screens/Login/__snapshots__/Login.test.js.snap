// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Screen structure should match the snapshot 1`] = `
<View>
  <Text>
    Login Screen
  </Text>
</View>
`;
